app/src/test/java/mollie/tictactoe/GameHelperTest.java

```diff
@@ -0,0 +1,143 @@
+package mollie.tictactoe;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import mollie.tictactoe.players.MobilePlayer;
+import ttt.Player;
+import ttt.game.Board;
+import ttt.game.GameEngine;
+import ttt.game.Marks;
+
+import static junit.framework.Assert.assertEquals;
+import static junit.framework.Assert.assertFalse;
+import static junit.framework.Assert.assertTrue;
+
+public class GameHelperTest {
+
+    private GameHelper mGameHelper;
+
+    @Before
+    public void setUp() {
+        mGameHelper = new GameHelper();
+    }
+
+    @Test
+    public void createsAGameWithTwoHumanPlayers() {
+        GameEngine game = mGameHelper.createGame();
+
+        Player playerOne = game.getCurrentPlayer();
+        game.play(1);
+        Player playerTwo = game.getCurrentPlayer();
+
+        assertEquals(MobilePlayer.class, playerOne.playerType());
```

Comment:
The playerType method here suggests you are setting up for type checks rather than polymorphism. If the method exists only for testing, I think you could get away with an `assertTrue(playerOne instanceOf MobilePlayer.class)`.